src/System.Reflection.MetadataLoadContext/src/System/Reflection/MetadataLoadContext.CoreAssembly.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection.TypeLoading;
+
+namespace System.Reflection
+{
+    public sealed partial class MetadataLoadContext
+    {
+        private static readonly string[] CoreNames = { "mscorlib", "System.Runtime", "netstandard" };
```

Comment:
I would make `System.Runtime` first to optimize the order for .NET Core.

Reply:
I was going to follow up on you based on our prior discussion. There is a comment in MetadataLoadContext.Api.cs that says
```
Note that "System.Runtime" is not an ideal core assembly as it excludes some of the interop-related pseudo-custom attribute types 
such as DllImportAttribute. However, it can serve if you have no interest in those attributes. The CustomAttributes api
will skip those attributes if the core assembly does not include the necessary types.
```

Reply:
Hmm. We could probe for those attributes using System.Runtime.InteropServices if not found in core assembly. 

I don't think we should just silently drop the attributes unless you have mscorlib facade available to stand in as core assembly.

Reply:
On the order, mscorlib first will succeed in common build scenarios because by default a facade to it is present on @(ReferencePath) and this API allows the core assembly to forward the primitives. Mind you, I don't know if the cost of following those forwards is worth avoiding.